.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+    - 'main'
+    - 'develop'
+
+env:
+  AWS_REGION: us-east-1
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  ClusterNamesCSV_prod: appsdev-prod
+  ClusterNamesCSV_dev: appsdev-stg
```

Comment:
Optional: Could you make this an environment variable instead, and set environment for job to branch?
See https://github.com/silinternational/app-monitoring-archiver/pull/51/files for example

Reply:
For this one piece, I don't think it's worth it. But I do like how it worked in app-monitoring-archiver.